correlation/src/main/java/com/cerner/beadledom/correlation/CorrelationIdContext.java

```diff
@@ -0,0 +1,51 @@
+package com.cerner.beadledom.correlation;
+
+import java.io.Closeable;
+import javax.annotation.Nullable;
+
+/**
+ * A context that provides access to the Correlation ID.
+ */
+public interface CorrelationIdContext {
+  /**
+   * Returns the current Correlation ID, {@code null} if no Correlation ID is in context.
+   */
+  @Nullable
+  String get();
```

Comment:
To me, this method sort of reads like I'm getting the context rather than the ID from the context.

Could there ever be other things on the context? Would it be helpful for `get`/`set` to be `getId`/`setId`?

Reply:
"ever"..maybe, but for right now, I just see us having the ID. I'm good with renaming it to getId/setId.